docs/install/install-extensions.md

```diff
@@ -0,0 +1,278 @@
+
+The following command installs the single-tenant Github source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-github" file="github.yaml" >}}
+```
+
+The single-tenant GitHub source creates one Knative service per GitHub source.
+
+The following command installs the multi-tenant GitHub source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-github" file="mt-github.yaml" >}}
+```
+
+The multi-tenant GitHub source creates only one Knative service handling all
+GitHub sources in the cluster. This source does not support logging or tracing
+configuration yet.
+
+To learn more about the Github source, try
+[our sample](../eventing/samples/github-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Camel-K Source" %}}
+
+The following command installs the Apache Camel-K Source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-camel" file="camel.yaml" >}}
+```
+
+To learn more about the Apache Camel-K source, try
+[our sample](../eventing/samples/apache-camel-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Apache Kafka Source" %}}
+
+The following command installs the Apache Kafka Source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-kafka" file="source.yaml" >}}
+```
+
+To learn more about the Apache Kafka source, try
+[our sample](../eventing/samples/kafka/source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="GCP Sources" %}}
+
+The following command installs the GCP Sources:
+
+```bash
+# This installs both the Sources and the Channel.
+kubectl apply -f {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
+```
+
+To learn more about the Cloud Pub/Sub source, try
+[our sample](../eventing/samples/cloud-pubsub-source/README.md).
+
+To learn more about the Cloud Storage source, try
+[our sample](../eventing/samples/cloud-storage-source/README.md).
+
+To learn more about the Cloud Scheduler source, try
+[our sample](../eventing/samples/cloud-scheduler-source/README.md).
+
+To learn more about the Cloud Audit Logs source, try
+[our sample](../eventing/samples/cloud-audit-logs-source/README.md).
+
+{{< /tab >}}
+
+{{% tab name="Apache CouchDB Source" %}}
+
+The following command installs the Apache CouchDB Source:
+
+```bash
+kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-couchdb" file="couchdb.yaml" >}}
+```
+
+To learn more about the Apache CouchDB source, read the [documentation](https://github.com/knative-sandbox/eventing-couchdb/blob/main/source/README.md).
+
+{{< /tab >}}
+
+{{% tab name="VMware Sources and Bindings" %}}
+
+The following command installs the VMware Sources and Bindings:
+
+```bash
+kubectl apply -f {{< artifact org="vmware-tanzu" repo="sources-for-knative" file="release.yaml" >}}
+```
+
+To learn more about the VMware sources and bindings, try
+[our samples](https://github.com/vmware-tanzu/sources-for-knative/tree/master/samples/README.md).
+
```

Comment:
be consistent with capitalizing, this is why I'd recommend everything lower case, e.g. sources, brokers, bindings, channels, etc.

Reply:
see my above comment about fixing style issues